site/docs/v1/tech/admin-guide/webauthn.adoc

```diff
@@ -0,0 +1,137 @@
+---
+layout: doc
+title: Configure WebAuthn
+description: Learn how to configure WebAuthn for FusionAuth
+navcategory: admin
+---
+
+include::docs/v1/tech/shared/_advanced-edition-blurb.adoc[]
+
+== Overview
+
+WebAuthn provides the ability for users to authenticate in their browser using the same method they use to unlock their device, like a biometric scan or PIN. WebAuthn has some big security benefits over the traditional username and password. Read on to learn how to configure WebAuthn in your FusionAuth instance.
+
+* <<What is WebAuthn>>
+* <<Configuring WebAuthn>>
+** <<Tenant Configuration>>
+*** <<Relying party Id>>
+*** <<Workflow settings>>
+** <<Application Configuration>>
+
+== What is WebAuthn
+
+WebAuthn is a W3C https://www.w3.org/TR/webauthn-2/[specification] that defines an API to create and use public-key credentials in web applications. Check out FusionAuth's link:/blog/2022/09/13/what-is-webauthn-why-do-you-care[WebAuthn blog post] for more information on the basics of WebAuthn and its security benefits.
+
+== Configuring WebAuthn
+
+You can enable WebAuthn in your FusionAuth instance with a few configuration changes. This section explains these configuration options and provides detail to help you choose the best options for your instance.
+
+=== Tenant Configuration
+The bulk of WebAuthn configuration happens at the tenant level, including Relying Party settings and authenticator attachment and user verification requirements for WebAuthn workflows. Use the WebAuthn tab on the add/edit tenant page to configure these options.
+
+image::core-concepts/tenant-configuration-webauthn-general.png[Tenant Configuration - WebAuthn,width=1200, role=top-cropped,bottom-cropped]
+
+The settings in this first section are applied to all applications in the tenant.
+
+[.api]
+[field]#Enabled#::
+This toggle must be enabled to use WebAuthn on this tenant.
+
+[field]#Relying party Id#::
+The Relying Party Id controls the scope of WebAuthn passkeys (i.e. which sites a WebAuthn passkey can be used to authenticate). More on this link:#relying-party-id[below].
+
+[field]#Relying party name#::
+The Relying Party name is a human-readable name that may be displayed by the browser or operating system during a WebAuthn ceremony. The value should be something that your users will recognize such as the name of your company or service. If the field is left blank, FusionAuth will default the value to [field]#Issuer# from the General tab during WebAuthn ceremonies.
```

Comment:
```suggestion
The Relying Party name is a human-readable name that may be displayed by the browser or operating system during a WebAuthn ceremony. The value should be something that your users will recognize such as the name of your company or service. If the field is left blank, FusionAuth will default the value to [field]#Issuer# from the [breadcrumb]#General# tab during WebAuthn ceremonies.
```

Because it is a navigation reference, I use [breadcrumb] styling.